### PR TITLE
Fix bug with Geom abline

### LINKED ISSUE #92: geom_abline doesn't work
As [noted here](https://github.com/tdhock/animint/pull/84#issuecomment-112899816), `geom_abline()` isn't working. Here is a mini-example:

```r
library(animint)
p <- qplot(wt, mpg, data = mtcars) + geom_abline(intercept = 20)
animint2dir(list(p = p))
```

### PR DESCRIPTION
**Pull request by kferris10**
Pretty sure that this should take care of all the problems with `geom_abline()` and close #92 

[See here for fixed example](http://bl.ocks.org/kferris10/raw/8cb4ee8562e204eac381/)

### CODE REVIEW

**Comment by cpsievert** [comment]
Looks good to me! +1 
cc @tdhock

File: R/animint.R [hunk 1]
```diff
@@ -94,7 +94,9 @@ parsePlot <- function(meta){
       ##    theme.pars$panel.grid then from theme.pars$line
       for(i in 1:length(pars)) {
         if(is.null(pars[[i]])) pars[[i]] <- 
-          if(!is.null(theme.pars$panel.grid[[i]])) {
+          if("element_blank" %in% class(theme.pars$panel.grid)) {
+            theme.pars$line[[i]]
+          } else if( !is.null(theme.pars$panel.grid[[i]]) ) {
```
Comment (kferris10) [inline R/animint.R]: I just tacked this change on to here because it's pretty small.  Let me know if this still looks okay.

Comment (tdhock) [inline R/animint.R]: can you please add a comment that explains what you are trying to do? ideally add a reference to the appropriate test.

Comment (kferris10) [inline R/animint.R]: What I want to do is say "if the parameter is NULL then look for it in `theme.pars$panel.grid`, if it's NULL there then look for it in `theme.pars$NULL`.

There's an error, however, if `$panel.grid` is an empty list - i.e. if it's created by `element_blank()`.
```
x <- list()
x[[1]]
Error in x[[1]] : subscript out of bounds
```

So I believe I first need to check whether it's specified as `element_blank()` or not.  But is there a better way to do this?

Comment (tdhock) [inline R/animint.R]: there is the `is.blank` helper function which maybe would help clarify the code https://github.com/tdhock/animint/blob/master/R/animint.R#L142

Comment (kferris10) [inline R/animint.R]: I think this is a better solution than my previous attempt.  While this generates an error
```
x <- list()
x[[1]]
Error in x[[1]] : subscript out of bounds
```
this returns `NULL` like I want
```
x[["colour"]]
NULL
```

**Comment by cpsievert** [comment]
Looks OK to me +1

**Comment by tdhock** [comment]
+1

**Pull request merged**